lambdatimes/src/components/Content/Card.js

```diff
@@ -1,19 +1,29 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 
 const Card = props => {
+  console.log(props)
```

Comment:
Note in the console log that you passed all the info on the post inside the post prop, so you can access everything off it with  props.post.<Whatever you want from the post object> I.e. props.post.headline etc